### PR TITLE
API to fetch nodes in the clusters

### PR DESCRIPTION
**Pull request by sridhargaddam**
Signed-Off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

**Comment by tpantelis** [comment]
So this fetches and globally stores the number of nodes. Can you explain a bit in the summary why this is needed and how will it be used?

**Comment by sridhargaddam** [comment]
> So this fetches and globally stores the number of nodes. Can you explain a bit in the summary why this is needed and how will it be used?

I explained the use-case in the actual PR where this API is used. Please see - https://github.com/submariner-io/submariner/pull/1575

**Review by tpantelis — APPROVED** [review]

File: test/e2e/framework/framework.go [hunk 1]
```diff
@@ -260,6 +260,20 @@ func DetectGlobalnet() {
 	})
 }
 
+func FetchClusterNodes() error {
```
Comment (mkolesnik) [inline test/e2e/framework/framework.go]: The name is a bit confusing as you're actually initializing number of nodes per cluster, I'd change it to something clearer

**Review by mkolesnik — CHANGES_REQUESTED** [review]

File: test/e2e/framework/framework.go (hunk 1 above)
Comment (sridhargaddam) [reply test/e2e/framework/framework.go]: Its called as part of init method as you can see here - https://github.com/submariner-io/submariner/pull/1575 
where it's updating the cluster nodes. What name would you prefer?

Comment (tpantelis) [reply test/e2e/framework/framework.go]:
```suggestion
func InitNumClusterNodes() error {
```

Comment (sridhargaddam) [reply test/e2e/framework/framework.go]: Done

**Review by mkolesnik — APPROVED** [review]

**Pull request merged**